itests/qtest/src/test/java/org/apache/hadoop/hive/cli/TestTezPerfCliDriver.java

```diff
@@ -34,7 +34,7 @@
 @RunWith(Parameterized.class)
 public class TestTezPerfCliDriver {
 
-  static CliAdapter adapter = new CliConfigs.TezPerfCliConfig(false).getCliAdapter();
+  static CliAdapter adapter = new CliConfigs.TezCustomTPCDSCliConfig(false).getCliAdapter();
```

Comment:
this is changing the naming contract between the configs and the perf

I think because we will have a new set of `PerfCliDriver`s we could just remove the old ones - or retain the conf names as well..

Reply:
Well in that case and if we keep the existing perf drivers (which it might make sense in the short term) it might be better to rename also the test class (e.g., `TestTezCustomTPCDSCliDriver`) to better reflect what we are really testing.

Reply:
you can do that - but I still don't see why do we need to add the keyword `custom` to the name of the test...it was around for a few years ...now we rename it and add "custom" to its name while there won't be a non-custom version anymore...

Reply:
Now we have `TestTezTPCDS30TBPerfCliDriver` and `TezTPCDS30TBCliConfig`. WDYT?